dump-format.mediawiki

```diff
@@ -0,0 +1,90 @@
+<pre>
+  NEP: TBD
+  Title: Unified Neo chain dump format
+  Author: Ekaterina Pavlova <ekt@nspcc.ru>, Roman Khimov <roman@nspcc.ru>
+  Type: Informational
+  Status: Draft
+  Created: 2025-02-14
+</pre>
+
+==Abstract==
+
+This proposal standardizes a single Neo chain dump format that unifies the “full” and “incremental”
+approaches into one consistent structure. It enables different Neo node implementations to remain
+compatible when producing or consuming chain dumps, allowing easy import or export of chain
+data without confusion over file naming or format variations.
+
+==Motivation==
+
+Neo nodes have historically used two chain dump formats: a full dump from block 0 (e.g., chain.acc)
+and an incremental dump from an arbitrary block (e.g., chain.X.acc). This dual approach complicates
+tooling and confuses operators (e.g., chain.0.acc is actually incremental). This NEP unifies both
+formats into one, making dumps interchangeable, simplifying import/restore, and reducing confusion.
+
+==Specification==
+
+===Dump Structure===
+
+<pre>
+dump_file
```

Comment:
We need at least short description, like it is done in
https://github.com/neo-project/proposals/blob/ff39b57eb67981b1ec7f0b244ec053aacdb7139a/nep-16.mediawiki?plain=1#L26